examples/prometheus/README.md

```diff
@@ -0,0 +1,208 @@
+# Getting Started with Prometheus and Grafana
+
+- [Export metrics from the application](#export-metrics-from-the-application)
+  - [Check results in the browser](#check-results-in-the-browser)
+- [Collect metrics using Prometheus](#collect-metrics-using-prometheus)
+  - [Configuration](#configuration)
+  - [Start Prometheus](#start-prometheus)
+  - [View results in Prometheus](#view-results-in-prometheus)
+- [Explore metrics using Grafana](#explore-metrics-using-grafana)
+- [Learn more](#learn-more)
+
+## Export metrics from the application
+
+It is highly recommended to go over the [ostream-metrics](../metrics_simple/README.md)
+doc before following along this document.
+
+Run the application with:
+
+```sh
+bazel run //examples/prometheus:prometheus_example
+```
+
+The main difference between the [ostream-metrics](../metrics_simple/README.md)
+example with this one is that the line below is replaced:
+
+```cpp
+std::unique_ptr<metric_sdk::MetricExporter> exporter{
+    new exportermetrics::OStreamMetricExporter};
+
+```
+
+with
+
+```cpp
+std::unique_ptr<metrics_sdk::MetricExporter> exporter{
+    new metrics_exporter::PrometheusExporter(opts)};
+```
+
+OpenTelemetry `PrometheusExporter` will export
+data via the endpoint defined by
+`metrics_exporter::PrometheusExporterOptions::url`,
+which is `http://localhost:8080/` by default.
```

Comment:
Not directly related to this PR, consider changing the default port following https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#prometheus-exporter (note that the spec is still Experimental).

Reply:
thanks, I fixed the port.